torch/ao/quantization/experimental/_gpu_quantization/smoothquant.py

```diff
@@ -0,0 +1,455 @@
+"""
+Testing out accuracy-only implementation of SmoothQuant
```

Comment:
As far as I know this isn't a fully faithful reimplement of smoothquant? Maybe we should revisit this later.